moveit_core/package.xml

```diff
@@ -72,5 +72,6 @@
     <build_type>ament_cmake</build_type>
     <moveit_core plugin="${prefix}/collision_detector_fcl_description.xml"/>
     <moveit_core plugin="${prefix}/collision_detector_bullet_description.xml"/>
+    <moveit_core plugin="${prefix}/filter_plugin_butterworth.xml"/>
```

Comment:
We don't need these exports in ROS2, it all happens in the CMake's pluginlib_export_plugin_description_file macro